### PR TITLE
FlowFuse 2.0 release blog post

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

The FlowFuse 2.0 Blog Announcement. I aimed to write the article short and understandably for those not familiar with FlowFuse, as one of the main reasons for the 2.0 release is also LeadGen.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+date: 2024-01-18
+authors: ["marian-demme"]
+#image: /blog/2024/01/images/
+tags:
+    - posts
+    - flowfuse
+    - news
+---
+
+Following the release of FlowFuse 1.0 end of 2022, we're excited to release FlowFuse 2.0, marking a significant step in managing Node-RED remote instances, which we term Devices.
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:14]:
```suggestion
Following the release of FlowFuse 1.0 end of 2022, we're excited to release FlowFuse 2.0, marking a significant step in managing Node-RED remote instances, which we call Devices. 
```

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Looking Ahead
+
+At FlowFuse, our mission is to empower bottom-up innovation and enable organizations to transform their workflows into business-critical applications with unprecedented efficiency. As we move forward, we are excited to invite our users to actively engage with our future developments. Our  [Roadmap](https://flowfuse.com/product/roadmap/) lays out the advancements we're targeting, offering a glimpse into the features and enhancements that are on the horizon. We also encourage our users to stay informed and involved by checking out our latest updates in our detailed [changelog](https://flowfuse.com/changelog/). Your insights and feedback are crucial to us; they fuel our commitment to continuous improvement and innovation. We warmly invite you to [share your thoughts and suggestions](https://flowfuse.com/contact-us/), as your input is a vital part of our journey in shaping the next steps for FlowFuse.
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:30]:
```suggestion
At FlowFuse, our mission is to empower bottom-up innovation and enable organizations to transform their workflows into business-critical applications with unprecedented efficiency. As we move forward, we are excited to invite our users to actively engage with our future developments. Our [Roadmap](https://flowfuse.com/product/roadmap/) lays out the advancements we're targeting, offering a glimpse into the features and enhancements that are on the horizon. We also encourage our users to stay informed and involved by checking out our latest updates in our detailed [changelog](https://flowfuse.com/changelog/). Your insights and feedback are crucial to us; they fuel our commitment to continuous improvement and innovation. We warmly invite you to [share your thoughts and suggestions](https://flowfuse.com/contact-us/), as your input is a vital part of our journey in shaping the next steps for FlowFuse.

## How to get started

<explain how FlowFuse run on-prem and in the cloud, link to on-prem install docs, and to the app to create an account>
```

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+tags:
+    - posts
+    - flowfuse
+    - news
+---
+
+Following the release of FlowFuse 1.0 end of 2022, we're excited to release FlowFuse 2.0, marking a significant step in managing Node-RED remote instances, which we term Devices. 
+<!--more-->
+
+Many organizations position Node-RED instances on remote servers, edge or industrial devices, to meet network, infrastructure, or use case requirements. Therefore, management of remote instances is crucial for the overall success of closing the gap between IT and OT. A key enhancement was the introduction of Device Groups (from version 1.15) and the new feature to assign target snapshots. This allows for direct and streamlined management of Node-RED Device fleets, setting the stage for future advancements in device management capabilities.
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:17]:
```suggestion
Many organizations position Node-RED instances on remote servers like edge or industrial devices. This way they can meet network requirement, interact with analog protocols, and overcome other infrastructure requirements. Management of remote instances is crucial for the overall success of closing the gap between IT and OT. A key enhancement was the introduction of Device Groups (from version 1.15) and the new feature to assign target snapshots. This allows for direct and streamlined management of Node-RED Device fleets, setting the stage for future advancements in device management capabilities.
```

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+FlowFuse is committed to augmenting the enterprise-readiness of Node-RED with introductions like [Single Sign-On (SSO)](https://flowfuse.com/docs/admin/sso/), [Multi-Factor Authentication (MFA)](https://flowfuse.com/docs/user/user-settings/#two-factor-authentication), and [High Availability](https://flowfuse.com/docs/user/high-availability/) since version 1.0. Furthermore, we recently achieved [SOC2 Type 1 compliance](https://flowfuse.com/blog/2024/01/soc2/). With these advancements, Node-RED, in combination with FlowFuse, is genuinely ready for enterprise and production use.
+
+## Enhanced Integration Capabilities
+The Node-RED Library has always been a cornerstone, offering over 4800 connectors (nodes) for various OT and IT protocols., thanks to the community and the Node-RED library. Building on this foundation, FlowFuse introduced "Certified Nodes" and "Blueprints". These [Blueprints](https://flowfuse.com/blog/2023/10/blueprints/) are designed to provide an easier start with Node-RED, showcasing its full potential, while Certified Nodes ensure the security of the nodes used. Learn more about our new Certified Nodes [here](https://flowfuse.com/blog/2023/10/certified-nodes/).
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:26]: Node-RED library is confusing for me, as I thought about the team library.

```suggestion
The Node-RED Flow Library has always been a cornerstone, offering over 4800 connectors (nodes) for various OT and IT protocols. Thanks to the community and the Node-RED library. Building on this foundation, FlowFuse introduced "Certified Nodes" and "Blueprints". These [Blueprints](https://flowfuse.com/blog/2023/10/blueprints/) are designed to provide an easier start with Node-RED, showcasing its full potential, while Certified Nodes ensure the security of the nodes used. Learn more about our new Certified Nodes [here](https://flowfuse.com/blog/2023/10/certified-nodes/).
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2024/01/flowfuse-release-2-0.md (hunk 2 above)
Comment (MarianRaphael) [reply src/blog/2024/01/flowfuse-release-2-0.md]: [How to get started](https://github.com/FlowFuse/website/pull/1526/commits/d10815a5fb951b3ac16024b34d6d798fbf53efb7)

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+image: /blog/2024/01/images/flowfuse-2-release.png
+tags:
+    - posts
+    - flowfuse
+    - news
+    - releases
+---
+
+Following the release of FlowFuse 1.0 end of 2022, we're excited to release FlowFuse 2.0, marking a significant step in managing Node-RED remote instances, which we call Devices. 
+<!--more-->
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:16]:
```suggestion
Following the release of FlowFuse 1.0 end of 2022, we're excited to release FlowFuse 2.0, marking a significant step in managing Node-RED remote instances, which we call Devices. FlowFuse already was the best place to operate Node-RED at scale in the cloud or on-premise, now it's able to manage Node-RED where ever it's run.
 
<!--more-->

```

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+For our FlowFuse users, this means it is no longer necessary or recommended to assign devices to an instance. Node-RED devices can be managed independently, and snapshots can be assigned via DevOps pipelines.
+
+## Enterprise-Readiness
+
+FlowFuse is committed to augmenting the enterprise-readiness of Node-RED with introductions like [Single Sign-On (SSO)](https://flowfuse.com/docs/admin/sso/), [Multi-Factor Authentication (MFA)](https://flowfuse.com/docs/user/user-settings/#two-factor-authentication), and [High Availability](https://flowfuse.com/docs/user/high-availability/) since version 1.0. Furthermore, we recently achieved [SOC2 Type 1 compliance](https://flowfuse.com/blog/2024/01/soc2/). With these advancements, Node-RED, in combination with FlowFuse, is genuinely ready for enterprise and production use.
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:24]: Nice line!

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+## Enhanced Integration Capabilities
+The Node-RED Flow Library has always been a cornerstone, offering over 4800 connectors (nodes) for various OT and IT protocols. Thanks to the community and the Node-RED library. Building on this foundation, FlowFuse introduced "Certified Nodes" and "Blueprints". These [Blueprints](https://flowfuse.com/blog/2023/10/blueprints/) are designed to provide an easier start with Node-RED, showcasing its full potential, while Certified Nodes ensure the security of the nodes used. Learn more about our new Certified Nodes [here](https://flowfuse.com/blog/2023/10/certified-nodes/).
+
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:28]:
```suggestion

## Developer Velocity

FlowFuse allowed teams to have access to the same Node-REDs, and collaborate with development. To speed up teams we've worked hard on maturing our [snapshot capabilties](todo) and build a full [DevOps Pipeline](todo). This provides team the option of developing their flows in a development instance, and have a QA review before any changes make it to production.

<todo chat about more dev velocity features>
```

File: src/blog/2024/01/flowfuse-release-2-0.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+## Looking Ahead
+
+At FlowFuse, our mission is to empower bottom-up innovation and enable organizations to transform their workflows into business-critical applications with unprecedented efficiency. As we move forward, we are excited to invite our users to actively engage with our future developments. Our  [Roadmap](https://flowfuse.com/product/roadmap/) lays out the advancements we're targeting, offering a glimpse into the features and enhancements that are on the horizon. We also encourage our users to stay informed and involved by checking out our latest updates in our detailed [changelog](https://flowfuse.com/changelog/). Your insights and feedback are crucial to us; they fuel our commitment to continuous improvement and innovation. We warmly invite you to [share your thoughts and suggestions](https://flowfuse.com/contact-us/), as your input is a vital part of our journey in shaping the next steps for FlowFuse.
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/flowfuse-release-2-0.md:31]:
```suggestion
At FlowFuse, our mission is to empower bottom-up innovation and enable organizations to transform their workflows into business-critical applications with unprecedented efficiency. As we move forward, we are excited to invite our users to actively engage with our future developments. Our [Roadmap](https://flowfuse.com/product/roadmap/) lays out the advancements we're targeting, offering a glimpse into the features and enhancements that are on the horizon. We also encourage our users to stay informed and involved by checking out our latest updates in our detailed [changelog](https://flowfuse.com/changelog/). Your insights and feedback are crucial to us; they fuel our commitment to continuous improvement and innovation. We warmly invite you to [share your thoughts and suggestions](https://flowfuse.com/contact-us/), as your input is a vital part of our journey in shaping the next steps for FlowFuse.
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2024/01/flowfuse-release-2-0.md (hunk 7 above)
Comment (MarianRaphael) [reply src/blog/2024/01/flowfuse-release-2-0.md:30]: How about @ZJvandeWeg :
```suggestion
## Developer Velocity

Node-RED team development is made possible with FlowFuse. Different development team members are able to share and collaborate on the same Node-RED instance. This makes for much easier collaboration between Node-RED developers.  We've worked hard on maturing our [snapshot capabilties](https://flowfuse.com/docs/user/snapshots/) and introduced  [DevOps Pipelines](https://flowfuse.com/docs/user/devops-pipelines/) that can be set up to stage Node-RED instances that have different development stages, e.g. test, development and production.
```

Comment (ZJvandeWeg) [reply src/blog/2024/01/flowfuse-release-2-0.md:30]: Double space before `[DevOps]`, otherwise it's great!

**Comment by ZJvandeWeg** [comment]
@MarianRaphael Merge when this is deployed to prod! :)

**Pull request merged**